diagrams/context.puml

```diff
@@ -73,12 +73,12 @@ services   <-R-> stores     : Query/Transact\n[in terms of events]
 stores       ..> publishers : are followed by
 stores       ..> reactors   : trigger
 
-reactors     --> services   : feed back into
+reactors     --> services   : feed data back into
```

Comment:
I'm intentionally being vague/general here - a reaction is driving a decision and/or could be triggering idempotent command processing driven by a process manager. I'm not sure what a better term for that might be, but _data_ seems overly specific and makes it sound more like an Ingester (which probably can benefit from being cast as doing something straightforward)

Reply:
Right now, I think it's too vague. Saying even feed decision back into the service would make this a lot clearer

Reply:
https://github.com/jet/equinox/blob/diag/DOCUMENTATION.md now says "manage follow-on reactions via"